splice/webapp.py

```diff
@@ -5,9 +5,14 @@
 register_flask_restful = False
 
 
-def setup_routes(app):
+def setup_routes(app, content_signing_only):
     global register_flask_restful
 
+    import splice.web.api.content
```

Comment:
better to put a if statement for each type of API endpoint being registered.

e.g.
```python
if 'tiles' in app.config.ALLOWED_APPS:
    """do tiles API registration"""

if 'signing' in app.config.ALLOWED_APPS:
    import splice.web.api.content
    splice.web.api.content.register_routes(app)
```

Much better than short-circuiting the function